wishful_upis/radio.py

```diff
@@ -24,6 +24,7 @@
 ]
 
 
+
 # Generic API to control the lower layers, i.e. key-value configuration.
 def set_parameters(param_key_values_dict):
```

Comment:
Op 7/27/2016 3:25 PM, NiltonDuarte schreef:
>
> In wishful_upis/radio.py 
> <https://github.com/wishful-project/wishful_upis/pull/1#discussion_r72437821>:
>
> >  @@ -24,6 +24,7 @@
> >   ]
> >
> >
> >  +
> >   # Generic API to control the lower layers, i.e. key-value configuration.
> >   def set_parameters(param_key_values_dict):
>
> This seens like a workaround on this issue 
> (wishful-project/framework#1 
> <https://github.com/wishful-project/framework/issues/1>).
> This upi does the same as set_parameter_lower_layer(**kwargs) 
> <https://github.com/wishful-project/wishful_upis/blob/master/wishful_upis/radio.py#L25>?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/wishful-project/wishful_upis/pull/1/files/361748734f36f8d54733de42b12678b4e988429d#r72437821>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ANXgT4k79mBId071ds1Bmqy7FJDy1tcTks5qZ1w_gaJpZM4JV__h>.
>
Indeed.

-- 
PhD Student Peter Ruckebusch
Department of Information Technology
Internet Based Communication Networks and Services (IBCN)
Ghent University - iMinds
Technologiepark-Zwijnaarde 15, B-9052 Zwijnaarde, Belgium
T: +32 (0)9 33 14981
T Secr: +32 (0)9 33 14900
E: peter.ruckebusch@intec.UGent.be
W : www.ibcn.intec.UGent.be